timeless/message/property_resource.py

```diff
@@ -15,5 +21,18 @@ class PropertyResource(MessageResource):
 
     """
 
+    directory = None
+    locale = None
+
+    def __init__(self, **kwargs):
+        self.directory = kwargs.get("directory")
+        self.locale = kwargs.get("locale")
+
     def get(self, key):
-        raise Exception("PropertyResources not implement yet!")
+        path = os.path.join(app.root_path, self.directory)
+        config = ConfigParser()
+        config.read(f"{path}message_{self.locale}.properties")
+        if not config.get("MESSAGES", key):
+            raise Exception("Value not found for key")
+        return config.get("MESSAGES", key)
+
```

Comment:
@paulodamaso can we chain the calls to ``config.read(...)`` and ``config.get(...)``? So it looks like this

```python
value = config.read(...).get("MESSAGES", key)
if not value 
    throw Exception(...)
return value
```

Reply:
@amihaiemil `config.read(...)` does not return anything, it just load the file, so we can't do the way you suggested :(